### PR TITLE
[Platform] [Accton] Validate mii-tool access on management port.

### PR DESCRIPTION
**Pull request by roylee123**
Add 2 kernel cmdline, tg3.short_preamble=1 tg3.bcm5718s_reset=1, to validate mdio acccess to external PHY.
For some accton platforms are equipped BCM5720 as MAC chip of their management port. 
Change kernel driver, tg3,  to support "short preamble" feature of BCM5720.

These change needs also kernel patching, https://github.com/Azure/sonic-linux-kernel/pull/71.

Signed-off-by: roy_lee <roy_lee@accton.com>

**- What I did**
Bring kernel cmdline to support short_preamble and bcm5718s_reset.
For accton platforms with ethernet controller, BCM5720.

Total 5 platforms:
as7116_54x, as7326_56x, as7716_32x-r0, as7716_32xb, and as7816_64x.

**- How I did it**
Add extra kernel cmdline to installer.conf on these platforms.

**- How to verify it**
Verified by mii-tool. (31 is the address of external phy).

root@sonic:/home/admin# mii-tool -v eth0 -p 31
using the specified MII index 31.
eth0: negotiated, link ok
  product info: vendor 00:d8:97, model 17 rev 2
  basic mode:   autonegotiation enabled
  basic status: autonegotiation complete, link ok
  capabilities: 1000baseT-HD 1000baseT-FD 100baseTx-FD 100baseTx-HD 10baseT-FD 10baseT-HD
  advertising:  flow-control
  link partner: 1000baseT-FD 100baseTx-FD 100baseTx-HD 10baseT-FD 10baseT-HD

### CODE REVIEW

**Comment by lguohan** [comment]
I have comments on the other PR.

**Comment by lguohan** [comment]
@roylee123 , can you update the kernel submodule in this PR?

**Comment by roylee123** [comment]
Hi, I've updated the kernel submodule. 
If I have to make the same change of this PR to branch 201807, should I sent another PR for it?
Or you will merge this PR to 201807?

File: device/accton/x86_64-accton_as7116_54x-r0/installer.conf [hunk 1]
```diff
@@ -1,4 +1,5 @@
 CONSOLE_PORT=0x3f8
 CONSOLE_DEV=0
 CONSOLE_SPEED=115200
-ONIE_PLATFORM_EXTRA_CMDLINE_LINUX="pci=noaer"
\ No newline at end of file
+ONIE_PLATFORM_EXTRA_CMDLINE_LINUX="pci=noaer"
+ONIE_PLATFORM_EXTRA_CMDLINE_LINUX="tg3.short_preamble=1 tg3.bcm5718s_reset=1"
```
Comment (lguohan) [inline device/accton/x86_64-accton_as7116_54x-r0/installer.conf]: should you just have one ONIE_PLATFORM_EXTRA_CMDLINE_LINUX? Are they conflicting to each other?

Comment (roylee123) [reply device/accton/x86_64-accton_as7116_54x-r0/installer.conf]: @lguohan, you are right. This platform is not even maintained by Accton. 
I should restore it back.

**Pull request merged**